android/pytorch_android/src/androidTest/java/org/pytorch/PytorchInstrumentedTests.java

```diff
@@ -298,6 +299,105 @@ public void testStr3Concat() throws IOException {
     }
   }
 
+  @Test
+  public void testAdd1ForByteTensors() throws IOException {
+    final long[] inputTensorShape = new long[]{256};
+    final long numElements = Tensor.numel(inputTensorShape);
+    final byte[] inputTensorData = new byte[(int) numElements];
+    for (int i = 0; i < numElements; ++i) {
+      inputTensorData[i] = (byte) i;
+    }
+    final Module module = Module.load(assetFilePath(TEST_MODULE_ASSET_NAME));
+
+    final Tensor inputTensorUint8 = Tensor.newUInt8Tensor(inputTensorShape, inputTensorData);
+    final Tensor inputTensorInt8 = Tensor.newInt8Tensor(inputTensorShape, inputTensorData);
```

Comment:
If you change this to

```
    final Tensor inputTensorUint8 = Tensor.newInt8Tensor(inputTensorShape, inputTensorData);
    final Tensor inputTensorInt8 = Tensor.newUint8Tensor(inputTensorShape, inputTensorData);
```

Does the test fail?  If so, how?

Reply:
It passes :) No difference on binary representation. 
I will think about replacement for this test to have the difference.